src/api/config/initializers/influxdb_custom_subscriber.rb

```diff
@@ -0,0 +1,4 @@
+if CONFIG['influxdb_hosts'].present?
+  require Rails.root.join('lib', 'influxdb_obs', 'influxdb_obs').to_s
```

Comment:
All files in `lib/` are autoloaded...

Reply:
Ah no? All code in app is autoloaded, lib is not autoloaded and either needs to be specified in application.rb or in the initializer.

https://stackoverflow.com/a/40019108